cmd/controller/main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 	logger, atomicLevel := logging.NewLoggerFromConfig(loggingConfig, logging.ControllerLogKey)
 	defer logger.Sync()
 
-	logger.Info("Starting the Pipeline Controller")
+	logger.Info("Starting the Build Controller")
 
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler()
```

Comment:
We need to add the creation of the new controller in the cmd similar to the taskrun
```
controllers := []*controller.Impl{
		// Pipeline Controllers
		taskrun.NewController(opt,
			taskRunInformer,
			taskInformer,
			buildInformer,
			buildTemplateInformer,
			clusterBuildTemplateInformer,
		),
	}
```